vmware_dvs/agent/firewalls/vcenter_firewall.py

```diff
@@ -26,6 +31,56 @@
 CONF = config.CONF
 
 
+def firewall_updater_loop(lo):
+    pq = port_queue(lo)
+    pq.port_updater_loop()
+    while True:
+        dvs, ports = pq.get_tasks(5)
+        if dvs and len(ports) > 0:
+            updater(dvs, ports)
+        else:
+            time.sleep(1)
+
+
+class port_queue(object):
```

Comment:
Regarding pep8 class names should normally use the CapWords convention. Please, change port_queue -> PortQueue